ui/app/templates/variables/path.hbs

```diff
@@ -0,0 +1,48 @@
+{{page-title "Secure Variables: " this.model.absolutePath}}
+{{#each this.breadcrumbs as |crumb|}}
+  <Breadcrumb @crumb={{crumb}} />
+{{/each}}
+  <section class="section">
+    <div class="toolbar">
+      <div class="toolbar-item is-right-aligned is-mobile-full-width">
+        <div class="button-bar">
+        {{!-- TODO: make sure qpNamespace persists to here --}}
+        {{#if (can "create variable" namespace=this.qpNamespace)}}
+          <LinkTo
+            @route="variables.new"
+            @query={{hash namespace=this.qpNamespace path=(concat this.model.absolutePath "/")}}
```

Comment:
`blocking`:  This breaks from an architectural pattern that I was hoping to persist from [RFC-130:  Namespaces as Identifiers](https://docs.google.com/document/d/1ssx1q0k1c3wYhSBS0DIY08njk3jfUdy2cODhgkC7c24/edit).

Is it out of the question to serialize namespaces into the identifier to avoid this pattern?

I could see this causing lurking bugs and confusion on how we handle namespaces and potentially preventing us from setting namespaces in the configuration as mentioned in the RFC comments.

Reply:
After a conversation: namespaces don't apply in the context of a .new page any more than they do for a .index page, so keeping this as is for now.